test/e2e/framework/networking_utils.go

```diff
@@ -218,8 +218,9 @@ func (config *NetworkingTestConfig) DialFromContainer(protocol, containerIP, tar
 // pod and confirm it doesn't show up as an endpoint.
 func (config *NetworkingTestConfig) DialFromNode(protocol, targetIP string, targetPort, maxTries, minTries int, expectedEps sets.String) {
 	var cmd string
+	// Make the command timeout after 1s (no matter what protocol it is).
 	if protocol == "udp" {
-		cmd = fmt.Sprintf("echo 'hostName' | timeout -t 3 nc -w 1 -u %s %d", targetIP, targetPort)
+		cmd = fmt.Sprintf("echo 'hostName' | timeout -t 1 nc -w 1 -u %s %d", targetIP, targetPort)
```

Comment:
Hmm - this is interesting. When I'm running this command it doesn't work for me:
```
wojtekt@kubemark-2000-master ~ $ echo 'hostName' | timeout -t 1 nc -w 1 -u 10.180.1.84 8081
timeout: invalid option -- 't'
Try 'timeout --help' for more information.
```

@bprashanth - any thoughts about it

BTW - it seems that this change broke it, but I don't really understand why...